views/meta.ejs

```diff
@@ -71,6 +72,15 @@
       editor.setTheme("ace/theme/monokai");
       editor.setReadOnly(true);
       editor.setFontSize(14);
-  </script>
+    </script>
+    
+  <%- include('partials/footer') %>
+
+    <style>
+      .footer {
+        display: none;
+      }
+    </style>
 </body>
-</html>
+
+</html>
```

Comment:
Don't like that you (or your editor) is removing the trailing newline in changed files. I really prefer to have them. Also my editors adds them automatically so next time I modify a file, it will get reverted again.